### PR TITLE
Update interop (pass 2)

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

**Review by genlu — APPROVED** [review]

File: src/Compilers/Extension/Roslyn.Compilers.Extension.csproj [hunk 1]
```diff
@@ -123,26 +123,8 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Manual exclusions for obsolete interop assemblies -->
-    <!--<PackageReference Include="EnvDTE" Version="$(EnvDTEVersion)" ExcludeAssets="all" />-->
```
Comment (genlu) [inline src/Compilers/Extension/Roslyn.Compilers.Extension.csproj:126]: Remove those properties from versions.props

Comment (sharwell) [reply src/Compilers/Extension/Roslyn.Compilers.Extension.csproj:126]: I'm removing all the ones I can, but this one cannot.

Comment (sharwell) [reply src/Compilers/Extension/Roslyn.Compilers.Extension.csproj:126]: OK now they are removed!

**Comment by sharwell** [comment]
@genlu I got a passing build 🎉

**Pull request merged**